packages/pn-personafisica-webapp/src/pages/__test__/AppStatus.page.test.tsx

```diff
@@ -4,14 +4,13 @@ import {
   AppResponseMessage,
   DowntimeStatus,
   formatDate,
-  formatTimeHHMM,
   KnownFunctionality,
   ResponseEventDispatcher,
 } from '@pagopa-pn/pn-commons';
 import { render } from '../../__test__/test-utils';
 import AppStatus from '../AppStatus.page';
 import { APP_STATUS_ACTIONS } from '../../redux/appStatus/actions';
-
+import { formatTime } from '../../../../pn-commons/src/utils';
```

Comment:
**beware** this import should be from `@pagopa-pn/pn-commons`!!

Reply:
do it